content/master/getting-started/app-compositions.md

```diff
@@ -0,0 +1,545 @@
+---
+title: App Compositions Quickstart
```

Comment:
```suggestion
title: App Quickstart
```

Nit: The other titles are e.g. "AWS Quickstart", "GCP Quickstart" - without "Composition".

"App Model Quickstart" might be good too.